Makefile

```diff
@@ -68,6 +78,7 @@ clean:
 	@$(MAKE) -C sdk/                                clean
 	@$(MAKE) -C psw/                                clean
 	@$(RM)   -r $(ROOT_DIR)/build
+	@$(RM)   -rf external/dcap_source
```

Comment:
`external/dcap_source` folder is part of the repo. Unstaged changes exist after `make clean`. Only remove  the contents inside the folder is enough.

Reply:
I just removed this line. Simply calling `rm -rf external/dcap_source/*` leaves `.git` behinds. Maybe we need something like `make -C external/dcap_source clean`?